CHANGELOG.md

```diff
@@ -15,23 +16,24 @@ Sections are used under each version as follows:
 
 ### Added
 
+- Documentation for the release process.
 - Added beta versions (via [BRAT]).
 - Added notifications for some failures.
 - Added CHANGELOG.
 
 ### Fixed
 
 - Add zettel command wasn't working.
-- Titles were to `# \n\n` by default when user didn't provide a title.
+- Titles were set to `# \n\n` by default when user didn't provide a title.
 - Ignore filter for getting zettels was configured incorrectly.
 
-## [1.1.0] - 2021/10/20
+## [1.1.0] - 2021-10-20
```

Comment:
I like this change to the format / format should be MM/DD/YYYY not YYYY/MM/DD but I agree that YYYY-MM-DD is better.